packages/kununu-utils/kununu-logger/index.js

```diff
@@ -1,13 +1,19 @@
-import {createLogger, transports, format} from 'winston';
+import {createLogger, format, transports as winston} from 'winston';
+
+import FingersCrossed from './fingers-crossed';
 
 const stringify = require('json-stringify-safe');
 
 const {timestamp, printf} = format;
 const getColorizedMessage = message => `\x1b[32m${message}\x1b[0m`;
 
+// It won't be logged when below this level
 const minimumLogLevel = process.env.MINIMUM_LOG_LEVEL || 'info';
+// Logs below this level won't be immediately outputted, but when this is reached
+const activationLogLevel = process.env.ACTIVATION_LOG_LEVEL || 'error';
```

Comment:
Should this defaults be configured in some other file? These default are both here and in `fingers-crossed`